README.md

```diff
@@ -238,6 +251,19 @@ array([[0.  , 0.75, 0.  , 0.  ],
 ```
 
 
+### The Best Policy Problems
+
+For making the best policy problems for your random walk you can easily:
+
+```pycon
+>>> states = [0, 1, 2]
+>>> trans = np.array([[1, 0, 0], [1/2, 0, 1/2], [0, 1, 0]])
+>>> rw = RandomWalk(states, trans, payoff=[0, 1, 4], cost=[1, 0, 2], discount=0.5)
+>>> rw.best_policy(stop_states=[0])
+{'continue': [], 'stop': [0, 1, 2]}
+```
+
+
 ## References			
 
```

Comment:
Correct format:
<blockquote>1- Lawler, Gregory F. Introduction to stochastic processes. Chapman and Hall/CRC, 2018.</blockquote>
<blockquote>2- <a href="https://markusfeng.com/projects/graph/">Markusfeng</a></blockquote>